src/main/java/org/jitsi/videobridge/BitrateController.java

```diff
@@ -685,12 +685,24 @@ private EndpointBitrateAllocation(
 
             // Initialize rates.
             rates = new long[encodings.length];
+            int optimal180Idx = 0;
```

Comment:
@saghul @bbaldino I followed your suggestion in https://github.com/jitsi/jitsi-videobridge/pull/435. Thanks!